guides/keras_cv/retina_net_overview.py

```diff
@@ -0,0 +1,275 @@
+"""
+Title: Train an Object Detection Model on Pascal VOC 2007 using KerasCV
+Author: [lukewood](https://lukewood.xyz)
+Date created: 2022/08/02
+Last modified: 2022/08/02
+Description: Use KerasCV to train a RetinaNet on Pascal VOC 2007.
+"""
+
+"""
+## Overview
+
+KerasCV offers a complete set of APIs to allow you to train your own state-of-the-art,
+production-grade object detection model.  These APIs include object detection specific
+data augmentation techniques, models, and COCO metrics.
+
+To get started, let's sort out all of our imports and define global configuration parameters.
+"""
+
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow import keras
+from tensorflow.keras import callbacks as callbacks_lib
+from tensorflow.keras import optimizers
+
+import keras_cv
+from keras_cv import bounding_box
+
+BATCH_SIZE = 8
+EPOCHS = 1
+
+"""
+## Data loading
+
+In this guide, we use the data-loading function: `keras_cv.loaders.pascal_voc.load()`.
```

Comment:
Why not use `keras_cv.datasets` for dataset loaders (consistent with `keras.datasets` for users already familiar with it)? "loader" is a very generic term, to be avoided (it could refer to anything)

Reply:
https://github.com/keras-team/keras-cv/pull/711/files